crates/autopilot/src/decoded_settlement.rs

```diff
@@ -1372,19 +1390,19 @@ mod tests {
         let auction_external_prices = BTreeMap::from([
             (
                 addr!("31429d1856ad1377a8a0079410b297e1a9e214c2"),
-                U256::from(1000000000000000000u128),
+                U256::from(13651729965787u128),
             ),
             (
                 addr!("d533a949740bb3306d119cc777fa900ba034cd52"),
-                U256::from(1000000000000000000u128),
+                U256::from(297240362652567u128),
             ),
             (
                 addr!("da816459f1ab5631232fe5e97a05bbbb94970c95"),
-                U256::from(1000000000000000000u128),
+                U256::from(490332962282438u128),
             ),
             (
                 addr!("fbeb78a723b8087fd2ea7ef1afec93d35e8bed42"),
-                U256::from(1000000000000000000u128),
+                U256::from(2812973786294373u128),
```

Comment:
How these values were calculated?

Reply:
These are external prices taken from solver competition entry for this tx